scripts/jenkins/qa_devstack.sh

```diff
@@ -24,7 +24,7 @@ function h_setup_base_repos {
         . /etc/os-release
 
         DIST_NAME=${NAME}
-        DIST_VERSION=${VERSION_ID}
+        DIST_VERSION=${VERSION}
```

Comment:
I think that `DIST_VERSION=${VERSION_ID}` was correct, for example for my openSUSE `VERSION="13.2 (Harlequin)"`